library/src/main/scala/scala/scalajs/js/BigInt.scala

```diff
@@ -0,0 +1,188 @@
+package scala.scalajs.js
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+
+/**
+ * A built-in object that provides a way to represent whole numbers larger than
+ * 2 ^ 53 - 1, which is the largest number JavaScript can reliably represent
+ * with the Number primitive.
+ *
+ * BigInt can be used for arbitrarily large integers.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+ */
+@js.native
+@JSGlobal
+class BigInt private[this] () extends js.Object {
+
+  def +(other: BigInt): BigInt = js.native
+  def *(other: BigInt): BigInt = js.native
+  def -(other: BigInt): BigInt = js.native
+  def **(other: BigInt): BigInt = js.native
```

Comment:
This operator won't work, because Scala.js does not recognize `**` as a JavaScript operator at the moment.

Reply:
Where is a place Scala.js handle JS operators ?

Reply:
That's quite complicated because it requires to change the IR. It spans the entire pipeline from the IR definition to the compiler backend to the optimizer to the emitter. It probably deserves a different PR anyway, so let's try to do everything else in this PR and leave that for later.

Reply:
I will leave it for later PR.